CHANGELOG.md

```diff
@@ -1,5 +1,56 @@
 # Change Log
 
+## Version 3.2.0 *(2020-04-02)*
+
+This is a new minor release with numerous internal changes.
+
+### Summary of changes
+
+* Chucker won't load the whole response into memory anymore, but will mutlicast it with the help of temporary files. It allows to avoid issues with OOM, like in this [issue](https://github.com/ChuckerTeam/chucker/issues/218).
+This change also allows to avoid problems with Chucker consuming responses, like reported in this [issue](https://github.com/ChuckerTeam/chucker/issues/242).
+* Added a red open padlock icon to clearly indicate HTTP requests in transactions list.
+* Added TLS info (version and cipher suite) into `Overview` tab.
+* Added ability to encode/decode URLs.
+* Added RTL support.
+* Switched from AsyncTasks to Kotlin coroutines.
+* Switched from Travis CI to Github Actions.
```

Comment:
Move this at the bottom or prepend with an `Infra:` label as this is the least impactful change from the client point of view.

Reply:
I think we can remove it at all.